iulianIachimov/src/test/java/generics/VideoTest.java

```diff
@@ -0,0 +1,29 @@
+package generics;
+
+import genericstask.model.Video;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertNull;
+
+public class VideoTest {
+
+  @Test
+  public void setSourceTestPos() throws Exception {
+    Video video = new Video("Singing in the rain", "Netflix");
+    video.setSource("Youtube");
+    assertThat("Youtube", equalTo(video.getSource()));
+  }
+
+  @Ignore
```

Comment:
with ignore annotation should come a proper message, why this test is ignored

Reply:
A comment message after @Ignore was added